src/foam/nanos/menu/VerticalMenu.js

```diff
@@ -30,7 +30,7 @@ foam.CLASS({
 
   css: `
   ^ input {
-    width: 220px;
+    width: 220px !important;
```

Comment:
Where is real offending width?

Reply:
SMEStyles.js has a .foam-u2-search-TextSearchView input sets the width and takes precedent. I'll check if setting the width to 220px looks ok.

Reply:
I figured out what the issue was, the SMEStyles.js was set after the StackView was set causing the funky css precedence.